rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,6 +16,8 @@ toc::[]
 [id="rosa-q4-2024_{context}"]
 === Q4 2024
 
+* **`VolumeDetachTimeout` configuration applied to machine pools for {hcp-title}.** The ROSA service is applying a `VolumeDetachTimeout` configuration of 5 minutes to all machine pools. This prevents issues with node deletion when volumes fail to detach. This only applies to {hcp-title}.
```

Comment:
ROSA service would read something like: Red Hat OpenShift Service on AWS Service
I also think if you rearrange you could omit the last bit about only applying to HCP.
Just a suggestion though. 

```suggestion
* **`VolumeDetachTimeout` configuration applied to machine pools for {hcp-title}.** {hcp-title} clusters now apply a `VolumeDetachTimeout` configuration of 5 minutes to all machine pools. This prevents issues with node deletion when volumes fail to detach. 
```